frontend/pages/policies/ManagePoliciesPage/components/CalendarEventPreviewModal/_styles.scss

```diff
@@ -78,3 +78,9 @@
     margin-top: 0; // Defer to new gap styling
   }
 }
+
+// Pop up preview immediately.
+.modal__background:has(.calendar-event-preview-modal) {
+  transition: none;
+  animation: none;
```

Comment:
Without this, there's a background flicker as the first modal's background disappears immediately and the preview modal's background animates.